### PR TITLE
New package: batmon-0.8

### PR DESCRIPTION
**Pull request by phossil**
Battery monitor for GNUstep.

### CODE REVIEW

File: srcpkgs/batmon/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'batmon'
+pkgname=batmon
+version=0.8
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
```
Comment (ericonr) [inline srcpkgs/batmon/template:9]: These `depends` should be detected automatically.

**Review by ericonr — COMMENTED** [review]
@abenson might be interested.

File: srcpkgs/batmon/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/batmon/template:9]: `gnustep-base` and `gnustep-gui` will, but `gnustep-back` won't.

**Comment by abenson** [comment]
Is there still interest in this package?

**Comment by phossil** [comment]
ive long since stopped using void unfortunately TuT

**Pull request closed without merging**